tests/test_integration/test_commands/test_deploy.py

```diff
@@ -42,6 +50,9 @@ def test_deploy_complete_org(cdf_tool_config: CDFToolConfig, build_dir: Path) ->
         verbose=True,
     )
 
+    changed_resources = get_changed_resources(cdf_tool_config, build_dir)
+    assert not changed_resources, "Redeploying the same resources should not change anything"
```

Comment:
This is the extension of the testing using the `complete_org` and `complete_org_alpha` to ensure we have at least one example for each resource.